.dockerignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 bower_components
 env
+media
```

Comment:
Just curious, is this working properly? Did you do sanity check for this?

Reply:
yes! I did a check for it & it works perfectly.